### ISSUE TITLE
extensions-samples/socketio/socketio-chat failres with NPE

### ISSUE THREAD

**Issue by ppolushkin** [Atmosphere/atmosphere-samples#39]
```java
ERROR org.atmosphere.socketio.cpr.SocketIOAtmosphereInterceptor - 
java.lang.NullPointerException
	at org.atmosphere.cache.SessionBroadcasterCache.retrieveFromCache(SessionBroadcasterCache.java:73)
	at org.atmosphere.cpr.DefaultBroadcaster.retrieveTrackedBroadcast(DefaultBroadcaster.java:1026)
	at org.atmosphere.cpr.DefaultBroadcaster.checkCachedAndPush(DefaultBroadcaster.java:931)
	at org.atmosphere.cpr.DefaultBroadcaster.cacheAndSuspend(DefaultBroadcaster.java:1417)
	at org.atmosphere.cpr.DefaultBroadcaster.addAtmosphereResource(DefaultBroadcaster.java:1385)
	at org.atmosphere.cpr.AtmosphereResourceImpl.suspend(AtmosphereResourceImpl.java:408)
	at org.atmosphere.socketio.transport.WebSocketTransport.handle(WebSocketTransport.java:91)
	at org.atmosphere.socketio.cpr.SocketIOAtmosphereInterceptor.inspect(SocketIOAtmosphereInterceptor.java:238)
	at org.atmosphere.cpr.AsynchronousProcessor.invokeInterceptors(AsynchronousProcessor.java:310)
	at org.atmosphere.cpr.AsynchronousProcessor.action(AsynchronousProcessor.java:175)
	at org.atmosphere.cpr.AsynchronousProcessor.suspended(AsynchronousProcessor.java:104)
	at org.atmosphere.container.Jetty9AsyncSupportWithWebSocket.service(Jetty9AsyncSupportWithWebSocket.java:180)
	at org.atmosphere.cpr.AtmosphereFramework.doCometSupport(AtmosphereFramework.java:2079)
	at org.atmosphere.websocket.DefaultWebSocketProcessor.dispatch(DefaultWebSocketProcessor.java:570)
	at org.atmosphere.websocket.DefaultWebSocketProcessor.open(DefaultWebSocketProcessor.java:215)
	at org.atmosphere.container.Jetty9WebSocketHandler.onWebSocketConnect(Jetty9WebSocketHandler.java:62)
	at org.eclipse.jetty.websocket.common.events.JettyListenerEventDriver.onConnect(JettyListenerEventDriver.java:87)
	at org.eclipse.jetty.websocket.common.events.AbstractEventDriver.openSession(AbstractEventDriver.java:220)
	at org.eclipse.jetty.websocket.common.WebSocketSession.open(WebSocketSession.java:406)
	at org.eclipse.jetty.websocket.server.WebSocketServerConnection.onOpen(WebSocketServerConnection.java:63)
	at org.eclipse.jetty.server.HttpConnection.completed(HttpConnection.java:339)
	at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:444)
	at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:248)
	at org.eclipse.jetty.io.AbstractConnection$2.run(AbstractConnection.java:540)
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:610)
	at org.eclipse.jetty.util.thread.QueuedThreadPool$3.run(QueuedThreadPool.java:539)
	at java.lang.Thread.run(Thread.java:745)
```

**Comment by jfarcand** [comment]
@ppolushkin Fixed in 2.3.0 SNAPSHOT

**Comment by ppolushkin** [comment]
thank's
